habitat-lab/habitat/tasks/rearrange/__init__.py

```diff
@@ -51,3 +52,9 @@ def _try_register_rearrange_task():
         HabitatSimActions.extend_action_space("empty")
     if not HabitatSimActions.has_action("rearrange_stop"):
         HabitatSimActions.extend_action_space("rearrange_stop")
+    if not HabitatSimActions.has_action("extend_arm"):
+        HabitatSimActions.extend_action_space("extend_arm")
+    if not HabitatSimActions.has_action("face_arm"):
+        HabitatSimActions.extend_action_space("face_arm")
+    if not HabitatSimActions.has_action("reset_joints"):
+        HabitatSimActions.extend_action_space("reset_joints")
```

Comment:
are all of these implemented on the robot side? are any missing?

Reply:
Yeah these were all implemented. We were just using different names (eg. manipulation_mode, navigation_mode). I have removed these now though. Only arm_action (for joints) and base control action are exposed